src/flexipages/NPSP_Get_Started.flexipage

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this page api name will get the NPSP namespace once part of the package, let's rename the file (not the tab) to just `GetStarted.flexipage`.